### DISCUSSION TITLE
lb_type error for load balancer attaching to VPC

### DISCUSSION THREAD

**Discussion by maheshkoheda** [learndevopsonline/learndevopsonline#1381] [Q&A] [answered]
@r-devops Hi raghu, for load balancer attaching to VPC- i see lb_type error upon make dev. where as it is same in your session-36 and no issues for your code. May i know where is my mistake? attached picture below.

<img width="457" alt="lb_type error for load balancer" src="https://github.com/learndevopsonline/learndevopsonline/assets/25144175/cafe6be0-21d3-4656-a3cc-67f540ce540b">

**Comment by sreedharm07** [comment, marked as answer]
hi @maheshkoheda 
here in variable you given lb-type but you declared lb_type. instead of underscore you have given hyphen.


![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/a415e3fe-e536-40be-94a7-cf22d17ef47f)
